dev-utils/src/commands/run_command.rs

```diff
@@ -0,0 +1,91 @@
+use failure::{format_err, Error};
+use std::fs;
+use std::process::Command;
+use std::{thread, time};
+#[derive(Debug)]
+pub struct RunCommand {
+    chainid: String,
+    app_hash: String,
+}
+
+impl RunCommand {
+    pub fn new() -> Self {
+        RunCommand {
+            chainid: "".to_string(),
+            app_hash: "".to_string(),
+        }
+    }
+
+    fn get_tendermint_filename(&self) -> String {
+        format!(
+            "{}/.tendermint/config/genesis.json",
+            dirs::home_dir().unwrap().to_str().unwrap()
```

Comment:
Should we add a command option to customize genesis file path?

Reply:
ok, i will add a path

Reply:
for activation
export TENDERMINT_HOME=~/.tendermint

for de-activation
unset TENDERMINT_HOME